includes/admin/class-admin.php

```diff
@@ -40,7 +40,7 @@ protected function __construct( $plugin_base_file ) {
 		$this->plugin_basename = plugin_basename( $plugin_base_file );
 
 		// Load plugin's textdomain (i.e. translations)
-		load_plugin_textdomain( 'hm-content-toc', false, $this->plugin_basename . '/languages' );
+		load_plugin_textdomain( 'hm-content-toc', false, $this->plugin_basename . '/languages/' );
 
 		// Setup strings used extensively throughout the class
 		$this->page_title = __( 'HM Content TOC Settings', 'hm-content-toc' );
```

Comment:
@johnbillion I wanted to double check if that string will be translated. It does appear after the call to `load_plugin_textdomain`. The whole `__construct()` will run on the `plugins_loaded` action. Is my understanding correct? That this translation is ok to leave there and it will be picked up?